_data/settings.yml

```diff
@@ -0,0 +1,23 @@
+identifier:
+  accessible_labels:
+    description: Agency description
+    gsa_link_logo: GSA homepage
+    identifier: Agency identifier
+    links: Important links
+    login_gov_logo: Login.gov logo
+    usa_gov: U.S. government information and services
+  disclaimer: An official website of the [General Services
```

Comment:
[thought] this line break is is a bit jarring. did the linter do this? i wasn't aware that we really enforced line limits (especially ones this short)

Reply:
I don't think there's a lint for this currently. That being said, we _could_ have one using the published [`@18f/identity-normalize-yaml`](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/normalize-yaml), which respects the local Prettier configuration (including [print width](https://prettier.io/docs/en/options.html#print-width)) and which we use in other static sites [like `identity-site`](https://github.com/GSA-TTS/identity-site/blob/ab27dfeb90459fddabe5258d32f90cb3b6adc3df/Makefile#L21-L23).

Reply:
(sorry, wrong text box.) I saw this format in `identity-site`, and thought this was a thing we do. Noted.